### ISSUE TITLE
Execution failed and CreateProcess Error after merging branch with Gradle

### ISSUE THREAD

**Issue by shittake** [nus-cs2103-AY2223S2/forum#46]
After merging the branch with Gradle, closing the IDEA project and deleting the .idea folder, before opening up the project again as mentioned in https://se-education.org/guides/tutorials/gradle.html, I firstly faced the issue where it says that my Duke class is outside the scope of my root. This was resolved when I did Files -> Project Structure -> right click on src to make it a source in Intellij. 

Now, I can right click the Duke class and run Duke.main() as per done previously, but I get a few more errors that I am unable to resolve. The first is that "A problem occurred starting process 'command '//wsl$/Ubuntu/usr/lib/jvm/java-11-openjdk-amd64/bin/java'' while the second is that "CreateProcess error=193, %1 is not a valid Win32 application". By Googling it, it seems that there is some incompatibility between the Windows version and the program. This only appeared after I tried merging the branch with Gradle in it. Any help will be appreciated!

**Comment by damithc** [comment]
Are you on Windows? In which case use the Windows environment fully rather than involve WSL?

**Comment by shittake** [comment]
I'm not too sure what that means - as shown in the screenshot, I click on the "Run Duke.main()" as per what I have done previously and only got this error after merging the branch  with Gradle, and deleting the .idea folder. Not too sure if deleting the .idea folder could have resulted in such issues.
![image](https://user-images.githubusercontent.com/40803540/214004395-fccc2db2-0aa8-4c88-9f8f-dc2a2715944b.png)

As of now, the error messages I get is:
![image](https://user-images.githubusercontent.com/40803540/214004760-f1cbb1c4-2af1-40e3-9b98-f04570b38bfb.png)

and 

![image](https://user-images.githubusercontent.com/40803540/214004782-c5312b3f-ee86-42c3-8224-c7c6cab49d27.png)

Wonder if anyone else face(d) the same issue too? Thank you.

**Comment by damithc** [comment]
@shittake These two error messages seem to suggest you are trying to use from Windows the Java version that is meant for WSL (which is a Linux OS running within Windows)

> As of now, the error messages I get is: ![image](https://user-images.githubusercontent.com/40803540/214004760-f1cbb1c4-2af1-40e3-9b98-f04570b38bfb.png)
> 
> and
> 
> ![image](https://user-images.githubusercontent.com/40803540/214004782-c5312b3f-ee86-42c3-8224-c7c6cab49d27.png)
> 
> Wonder if anyone else face(d) the same issue too? Thank you.

Perhaps give https://github.com/nus-cs2103-AY2223S2/forum/issues/43#issuecomment-1399818661 a try?

**Comment by shittake** [comment]
Thanks for your reply. I have managed to resolve this issue. For those who are facing similar issues, this is the workflow that I followed:
1. Delete my entire project folder, and re-clone from the github repository
2. Import the project once again in Intellij, and merge the branch with the gradle support
3. In Intellij, click File --> Project Settings --> Project --> Select the appropriate SDK
4. In Intellij, File --> Project Settings --> Modules --> Right-click src folder and select it as a source. Under src --> main --> java, right-click this java folder and select it as a source as well.

Issue is settled after performing the above.
